external/anomaly/tools/sample.py

```diff
@@ -0,0 +1,242 @@
+"""
```

Comment:
Is it needed?
If yes, it should be tested somehow, I afraid that it might become non-working and we won't know about it.

Reply:
@Ilya-Krylov Does ote_cli support the anomaly detection task?

Reply:
anomaly classification - yes

Reply:
https://github.com/openvinotoolkit/training_extensions/blob/develop/tests/ote_cli/test_ote_cli_tools_anomaly_classification.py

Reply:
except compression, because it has not been merged yet.

Reply:
ok, I will check ote_cli and remove the sample at all.